otx/algorithms/common/tasks/training_base.py

```diff
@@ -234,7 +235,10 @@ def _initialize(self, export=False):
         else:
             self._recipe_cfg.pop("adaptive_validation_interval", None)
 
-        self.set_early_stopping_hook()
+        # TODO (sungchul): it will be removed after an update that applies hparam.yaml
```

Comment:
Maybe in the current status, hparam.yaml is applied. How about deleting?

Reply:
Could you explain where hparam.yaml is applied?

Reply:
Ah then, this file is not used ?
![image](https://user-images.githubusercontent.com/80735344/209253576-8bdc2ec3-a550-4f1a-89b9-4e2ab0a4d9c2.png)

Reply:
hparam.yaml is used when otx builder build workspace. Builder will modify base template, which is oriented to incremental learning, to match own training type

Reply:
Yeah. It is not used as of now. That otx builder which is not currently supported fully will adaptively overwrite the config from hparam.yaml. I said to Sungchul to leave configs as is, expecting them to process when otx build is supported. But seems like just unsetting the early stopping is needed. I think it's okay to good to leave this codes.

Reply:
If so, hparam.yaml is already applied to change the template. So, we can delete the (TODO) tag

Reply:
Ah I think TODO tag is needed because hparam support to disable early stopping too. Self-sl needs to ensure to run some training epochs so I added comments to leave as is.

Reply:
And this should be revisited. If we move self.task_type, which is in the each algo's inferencer, to this training_base, we can use it in this block

Reply:
Maybe I missed something..! Okay then leave it as is